eng/scripts/Language-Settings.ps1

```diff
@@ -95,10 +95,10 @@ function Get-java-PackageInfoFromPackageFile ($pkg, $workingDirectory) {
 function Publish-java-GithubIODocs ()
 {
   $PublishedDocs = Get-ChildItem "$DocLocation" | Where-Object -FilterScript {$_.Name.EndsWith("-javadoc.jar")}
```

Comment:
Need to also update PackageInfoFromPackageFile  to add ReleaseTag

Reply:
I'll do that when am focusing on that part of the code. There is the chance that it changes before I get to work on it. Right now I want to get the copy to blobs part done